src/index.js

```diff
@@ -51,17 +51,35 @@ class App {
     this.container.appendChild(addTask);
     this.container.appendChild(this.taskListContainer);
     this.container.appendChild(buttonDeleteCompletedTasks);
+
+    this.taskList = this.getTaskListFromLocalStorage();
+    this.render();
+  }
+
+  saveTaskListToLocalStorage() {
+    const taskListStr = JSON.stringify(this.taskList);
+    localStorage.setItem('toDoTaskList', taskListStr);
   }
 
+  getTaskListFromLocalStorage() {
+    const dataFromLocalStorage = localStorage.getItem(this.localStorageKey);
+    if (dataFromLocalStorage && dataFromLocalStorage.length) {
+      this.taskList = JSON.parse(dataFromLocalStorage);
+      return this.taskList;
+    }
+    return [];
+  }
```

Comment:
use this.taskList.push(data from local storage)

Reply:
done

Reply:
not needed